lib/tests/streamlit/pydeck_test.py

```diff
@@ -13,6 +13,8 @@
 # limitations under the License.
 
 import json
+import unittest
+import unittest.mock
```

Comment:
nit: not very important, but I think this is more frequently imported as `from unittest import mock` throughout our tests